docs/Admin.md

```diff
@@ -664,6 +579,81 @@ function AsyncResources() {
 }
 ```
 
+## Adding Custom Pages
+
+To register your own routes, pass one or several `<CustomRoutes>` elements as children of `<Admin>`. Declare as many [react-router-dom](https://reacttraining.com/react-router/web/guides/quick-start) `<Route>` as you want inside them:
```

Comment:
(https://reacttraining.com/react-router/web/guides/quick-start) leads to v5 website
did you mean something like (https://reactrouter.com/docs/en/v6) ?